pages/_app.js

```diff
@@ -7,21 +7,42 @@ function MyApp({ Component, pageProps }) {
 
     {}
   );
+  const [spendingValues, setSpendingValues] = useLocalStorage(
+    "spendingValues",
+
+    []
+  );
 
   function addBankAccount(bankAccount) {
     setBankAccount((previousBankAccount) => {
       return {
         ...previousBankAccount,
         name: bankAccount.name,
-        kontostand: bankAccount.kontostand,
+        kontostand: parseFloat(bankAccount.kontostand),
       };
     });
   }
 
+  function addSpendingValues(spendingValues) {
```

Comment:
I find it unusual that you declare functions here in App.js. Because normally only the components are rendered here (and in the Capstone with Michael's hook the Local Storage is integrated). Is there a reason for this? Shouldn't they better be in the index.js or on the corresponding page, where the function is also used? Maybe you know more than I do at this point 😃